### PR TITLE
Tweak few more strings re mixing and balances

### PR DESCRIPTION
**Pull request by UdjinM6**
A follow-up for #3211

### CODE REVIEW

**Comment by UdjinM6** [comment]
How about swapping it with "PrivateSend" (and tweaking strings accordingly in rpc and options)?

pinging @strophy

**Comment by strophy** [comment]
I would support using "PrivateSend balance" as a replacement for "fully mixed". I would suggest removing the definite article from "The desirable PrivateSend balance in DASH", maybe something more like "Target PrivateSend balance" is sufficient?

File: src/init.cpp [hunk 1]
```diff
@@ -610,7 +610,7 @@ std::string HelpMessage(HelpMessageMode mode)
     strUsage += HelpMessageOpt("-privatesendmultisession", strprintf(_("Enable multiple PrivateSend mixing sessions per block, experimental (0-1, default: %u)"), DEFAULT_PRIVATESEND_MULTISESSION));
     strUsage += HelpMessageOpt("-privatesendsessions=<n>", strprintf(_("Use N separate masternodes in parallel to mix funds (%u-%u, default: %u)"), MIN_PRIVATESEND_SESSIONS, MAX_PRIVATESEND_SESSIONS, DEFAULT_PRIVATESEND_SESSIONS));
     strUsage += HelpMessageOpt("-privatesendrounds=<n>", strprintf(_("Use N separate masternodes for each denominated input to mix funds (%u-%u, default: %u)"), MIN_PRIVATESEND_ROUNDS, MAX_PRIVATESEND_ROUNDS, DEFAULT_PRIVATESEND_ROUNDS));
-    strUsage += HelpMessageOpt("-privatesendamount=<n>", strprintf(_("Keep N DASH mixed (%u-%u, default: %u)"), MIN_PRIVATESEND_AMOUNT, MAX_PRIVATESEND_AMOUNT, DEFAULT_PRIVATESEND_AMOUNT));
+    strUsage += HelpMessageOpt("-privatesendamount=<n>", strprintf(_("The desirable PrivateSend balance in DASH (%u-%u, default: %u)"), MIN_PRIVATESEND_AMOUNT, MAX_PRIVATESEND_AMOUNT, DEFAULT_PRIVATESEND_AMOUNT));
```
Comment (PastaPastaPasta) [inline src/init.cpp]:
```suggestion
    strUsage += HelpMessageOpt("-privatesendamount=<n>", strprintf(_("Target PrivateSend balance (%u-%u, default: %u)"), MIN_PRIVATESEND_AMOUNT, MAX_PRIVATESEND_AMOUNT, DEFAULT_PRIVATESEND_AMOUNT));
```

File: src/qt/forms/optionsdialog.ui [hunk 2]
```diff
@@ -263,7 +263,7 @@
             <string notr="true"/>
            </property>
            <property name="text">
-            <string>Amount of Dash to keep mixed</string>
+            <string>The desirable PrivateSend balance</string>
```
Comment (PastaPastaPasta) [inline src/qt/forms/optionsdialog.ui]:
```suggestion
            <string>Target PrivateSend balance</string>
```

File: src/qt/overviewpage.cpp [hunk 3]
```diff
@@ -428,8 +428,8 @@ void OverviewPage::updatePrivateSendProgress()
 
     QString strToolPip = ("<b>" + tr("Overall progress") + ": %1%</b><br/>" +
                           tr("Denominated") + ": %2%<br/>" +
-                          tr("Mixed") + ": %3%<br/>" +
-                          tr("Anonymized") + ": %4%<br/>" +
+                          tr("Partially mixed") + ": %3%<br/>" +
+                          tr("Ready") + ": %4%<br/>" +
```
Comment (PastaPastaPasta) [inline src/qt/overviewpage.cpp]:
```suggestion
                          tr("Mixed") + ": %4%<br/>" +
```
Idk about this one... maybe "ready" is okay, it just sounds a little weird

File: src/rpc/misc.cpp [hunk 4]
```diff
@@ -64,7 +64,7 @@ UniValue getinfo(const JSONRPCRequest& request)
             "  \"protocolversion\": xxxxx,   (numeric) the protocol version\n"
             "  \"walletversion\": xxxxx,     (numeric) the wallet version\n"
             "  \"balance\": xxxxxxx,         (numeric) the total dash balance of the wallet\n"
-            "  \"privatesend_balance\": xxxxxx, (numeric) the mixed dash balance of the wallet\n"
+            "  \"privatesend_balance\": xxxxxx, (numeric) the PrivateSend dash balance of the wallet in " + CURRENCY_UNIT + "\n"
```
Comment (PastaPastaPasta) [inline src/rpc/misc.cpp]:
```suggestion
            "  \"privatesend_balance\": xxxxxx, (numeric) the PrivateSend balance in " + CURRENCY_UNIT + "\n"
```

File: src/rpc/privatesend.cpp [hunk 5]
```diff
@@ -93,7 +93,7 @@ UniValue getprivatesendinfo(const JSONRPCRequest& request)
                 "  \"multisession\": true|false,        (bool) Whether PrivateSend Multisession option is enabled\n"
                 "  \"max_sessions\": xxx,               (numeric) How many parallel mixing sessions can there be at once\n"
                 "  \"max_rounds\": xxx,                 (numeric) How many rounds to mix\n"
-                "  \"max_amount\": xxx,                 (numeric) How many " + CURRENCY_UNIT + " to keep mixed\n"
+                "  \"max_amount\": xxx,                 (numeric) The desirable PrivateSend balance of the wallet in " + CURRENCY_UNIT + "\n"
```
Comment (PastaPastaPasta) [inline src/rpc/privatesend.cpp]:
```suggestion
                "  \"max_amount\": xxx,                 (numeric) Target PrivateSend balance in " + CURRENCY_UNIT + "\n"
```

File: src/wallet/rpcwallet.cpp [hunk 6]
```diff
@@ -2534,7 +2534,7 @@ UniValue getwalletinfo(const JSONRPCRequest& request)
             "  \"walletname\": xxxxx,             (string) the wallet name\n"
             "  \"walletversion\": xxxxx,     (numeric) the wallet version\n"
             "  \"balance\": xxxxxxx,         (numeric) the total confirmed balance of the wallet in " + CURRENCY_UNIT + "\n"
-            "  \"privatesend_balance\": xxxxxx, (numeric) the mixed dash balance of the wallet in " + CURRENCY_UNIT + "\n"
+            "  \"privatesend_balance\": xxxxxx, (numeric) the PrivateSend dash balance of the wallet in " + CURRENCY_UNIT + "\n"
```
Comment (PastaPastaPasta) [inline src/wallet/rpcwallet.cpp]:
```suggestion
            "  \"privatesend_balance\": xxxxxx, (numeric) the PrivateSend balance in " + CURRENCY_UNIT + "\n"
```

**Pull request merged**